dev-tools/src/main/database/all_delete.sql

```diff
@@ -18,15 +18,15 @@ DELETE FROM atht_access_token;
 
 DELETE FROM athz_role WHERE NOT (scope_id = 1 AND id = 1);
 
-DELETE FROM athz_role_permission WHERE NOT (scope_id = 1 AND id = 1);
+DELETE FROM athz_role_permission WHERE NOT (scope_id = 1 AND role_id = 1 AND id IN (1,2,3,4,5,6,7,8,9,10,11,12,13,14,15));
```

Comment:
I introduced this script. I use it during tests so that database is cleaned of test created data.
Only basic seeded data is left in database. In case of some gherkin scenarios i don't want to create database from scratch using liquibase. It is so sloooow. But it is not meant for enithing more than testing.

But maybe this script should be moved into test module. What do you think?